src/Request/BaseRequest.php

```diff
@@ -103,7 +103,7 @@ public function sendRequest()
     protected function requestOptions()
     {
         $opts = [
-            'verify' => __DIR__ . '/../../resources/cert.pem',
+            'verify' => true,
```

Comment:
Maybe so! I'd be fine with using the cert provided in the package, but it just wasn't working in my environment.